dspy/clients/lm.py

```diff
@@ -334,32 +329,11 @@ def litellm_completion(request: Dict[str, Any], num_retries: int, cache={"no-cac
         max_retries=0,
     )
 
-    stream = dspy.settings.send_stream
-    if stream is None:
-        return litellm.completion(
-            cache=cache,
-            **retry_kwargs,
-            **request,
-        )
-
-    # The stream is already opened, and will be closed by the caller.
-    stream = cast(MemoryObjectSendStream, stream)
-
-    @syncify
-    async def stream_completion():
-        response = await litellm.acompletion(
-            cache=cache,
-            stream=True,
-            **retry_kwargs,
-            **request,
-        )
-        chunks = []
-        async for chunk in response:
-            chunks.append(chunk)
-            await stream.send(chunk)
-        return litellm.stream_chunk_builder(chunks)
-
-    return stream_completion()
+    return litellm.completion(
```

Comment:
Will this break backward compatibility on current streaming usage?

Reply:
discussed offline, will not delete the current logic. We will think about improvements in a followup PR (no timeline promise tho)